packages/core/src/shared/telemetry/vscodeTelemetry.json

```
@@ -480,6 +480,26 @@
             "name": "stepfunctions_previewstatemachine",
             "description": ""
         },
+        {
+            "name": "workflowStudio_opened",
+            "description": "Called after opening asl file in Step Functions Workflow Studio",
+            "metadata": [
+                {
+                    "type": "id",
+                    "required": true
+                }
+            ]
+        },
+        {
+            "name": "workflowStudio_closed",
```

is this not specific to SFN? use the existing `stepfunctions_` prefix if so. same for the other metric.

also, after the underscore, the first word is typically a "verb".

```suggestion
            "name": "stepfunctions_closeWorkflowStudio",
```